dotcom-rendering/src/model/newsletterIllustrations.ts

```diff
@@ -0,0 +1,101 @@
+	'us-morning-newsletter':
+		'https://i.guim.co.uk/img/media/035de4d15b7732659f574547e7cc68199c29432a/0_0_5000_3000/500.jpg?width=250&quality=45&s=5319289e84bf8248a3b75de85b947401',
+	'morning-mail':
+		'https://i.guim.co.uk/img/media/4cf911dd7c5cf23e4c67f0951ec39dc99f6cbc9a/0_0_1240_744/500.jpg?width=250&quality=45&s=238c865b55b4a60818b09b70f709303f',
+	'afternoon-update':
+		'https://i.guim.co.uk/img/media/50b02d4e4a32def95d8d26cc852549e6bd83f037/0_0_1850_1111/500.jpg?width=250&quality=45&s=6325bb62e51f4af4d43e8bafb28b3e6d',
+	'green-light':
+		'https://i.guim.co.uk/img/media/bf9a946c2bdd8011c098dfa9e8c91a94d033170f/0_0_1000_600/500.jpg?width=250&quality=45&s=28351eec6427ae3f8a4e6dc64ac38007',
+	'tech-scape':
+		'https://i.guim.co.uk/img/media/e4b75f936694609cff24a4a440ec07a87bc2e274/8_0_2258_1356/500.jpg?width=250&quality=45&s=36fa8941658accc6cdb5a481a779c087',
+	'the-rural-network':
+		'https://i.guim.co.uk/img/media/23ab4cead66f6e7fcf85b2b0f97faa9fae606858/0_275_4288_2573/500.jpg?width=250&quality=45&s=2d6e3dea809f9019ac7272cc3e5c4921',
+	'today-uk':
+		'https://i.guim.co.uk/img/media/b1e1d07a6c5645eff768afb0c2dc5eb915817cf0/1_0_5021_3014/500.png?width=250&quality=45&s=a82d74a2904598782d5dd249d860242c',
+	'today-us':
+		'https://i.guim.co.uk/img/media/d30b5b560d10e8bd429c848a3a6668787d88f716/2_0_5434_3262/500.png?width=250&quality=45&s=d6a46052173f521aa3986136c428163d',
+	'today-au':
+		'https://i.guim.co.uk/img/media/c95f590baa23c75de13686356f12ad0b59575a72/2_0_6221_3734/500.png?width=250&quality=45&s=82f88f7ce3715ab1dccde4bd4063d323',
+	'australian-politics':
+		'https://i.guim.co.uk/img/media/f55f510293ee713ffcc9e48d1f420d4b2769ea49/0_0_5472_3648/500.jpg?width=250&quality=45&s=fa2be1b3db7ba2065959f0e3babef546',
+	'global-dispatch':
+		'https://i.guim.co.uk/img/media/28ffd9cfbf7125265a79a664afacea6444c19cf1/0_0_2560_1536/500.jpg?width=250&quality=45&s=58da4319f10510a0b374324a928b766b',
+	'business-today':
+		'https://i.guim.co.uk/img/media/aa8b0d33b6d0c2ff8fa26f15cd42632d8a251a66/0_151_3000_1800/500.jpg?width=250&quality=45&s=e5d8298adae28ef97fbae18cde3f548b',
+	'minute-us':
+		'https://i.guim.co.uk/img/media/dc41d329183de03943d483df5e68f91a0f263a4e/0_0_5000_3000/500.jpg?width=250&quality=45&s=419ffb0a03b5f5c9cef62cd80c52053e',
+	'best-of-opinion':
+		'https://i.guim.co.uk/img/media/4ef30ca444a6980ad09f9c651b620000ede91d68/2943_7_2997_1798/500.png?width=250&quality=45&s=d403a4598a34e4c705513e157b4cdefb',
+	'best-of-opinion-us':
+		'https://i.guim.co.uk/img/media/4ef30ca444a6980ad09f9c651b620000ede91d68/3623_5_3289_1976/500.png?width=250&quality=45&s=698ab2c29ad2f9163683dbeeb7990f18',
+	'best-of-opinion-au': '',
+	'this-is-europe':
+		'https://i.guim.co.uk/img/media/0f029b430f0ce52d3e675b66dcfd7e9b86bf2b9b/0_1_1250_750/500.jpg?width=250&quality=45&s=971ab7ce3906642fa0582864443d3b06',
+	patriarchy:
```

Comment:
```suggestion
	'patriarchy':
```

Reply:
I'd tend to agree here, but the [guardian prettier rule](https://github.com/guardian/csnx/blob/8a1e22c413d07c522dead9af23478264f9361acb/libs/%40guardian/prettier/index.js#L7C26-L7C26) only allow quotes for object keys 'as needed'.